### DISCUSSION TITLE
TImeline into Datatable

### DISCUSSION THREAD

**Discussion by tgarnaldos** [primefaces/community#248] [PrimeFaces]
Hello, I'm trying to display a table with elements that, in addition to their regular attributes like the element's name and code, include a Timeline element. However, when I iterate through them in a Datatable, it only displays the timeline of the last element, but it shows the other attributes. I'm using PrimeFaces 12.

Thanks

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.
